pos_second_uom/static/src/js/add_button.js

```diff
@@ -0,0 +1,141 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
+import { ControlButtons } from "@point_of_sale/app/screens/product_screen/control_buttons/control_buttons";
+import { Dialog } from "@web/core/dialog/dialog";
+import { Component, onWillStart, useState, useEffect } from "@odoo/owl";
+import { usePos } from "@point_of_sale/app/store/pos_hook";
```

Comment:
```suggestion
import { Dialog } from "@web/core/dialog/dialog";
import { patch } from "@web/core/utils/patch";

import { usePos } from "@point_of_sale/app/store/pos_hook";
import { ControlButtons } from "@point_of_sale/app/screens/product_screen/control_buttons/control_buttons";

import { Component, onWillStart, useState, useEffect } from "@odoo/owl";
```

Grouping it makes the imports more readable.